lib/plugins/aws/package/compile/events/stream.js

```diff
@@ -9,6 +9,19 @@ class AwsCompileStreamEvents {
     this.provider = this.serverless.getProvider('aws');
 
     this.hooks = {
+      'initialize': () => {
+        if (
```

Comment:
I think this deprecation does not address the issue that we would like to address with the message. We don't really care about people who have this setting turned on, but rather about people that will be affected by the change with the next major as it might be a breaking change, so instead, we should check for people that have `consumer` set but do not use `kinesis.consumerNamingMode` that in v3 the naming scheme will change and they can already opt in by setting `kinesis.consumerNamingMode`.

Reply:
I agree, I'll have to redo this